docs/components/tracked-controls.md

```diff
@@ -0,0 +1,43 @@
+---
+title: tracked-controls
+type: components
+layout: docs
+parent_section: components
+---
+
+The tracked-controls component interfaces with tracked controllers. 
+Uses the Gamepad API to handle tracked controllers. 
+Selects the appropriate controller, applies pose to the entity, observes buttons state and emits appropriate events.
+
+
+## Example
+
+```html
+<a-entity tracked-controls="controller: 0; id: 'OpenVR Gamepad'"></a-entity>
```

Comment:
@Zeouterlimits thanks for taking the time. I believe this is the last piece of feedback to be adressed